src/bun.js/bindings/headers-handwritten.h

```diff
@@ -158,7 +154,7 @@ typedef struct ZigStackFrame {
 
 typedef struct ZigStackTrace {
     BunString* source_lines_ptr;
-    int32_t* source_lines_numbers;
+    OrdinalNumber* source_lines_numbers;
```

Comment:
We can't share C++ classes in Zig
```suggestion
    int32_t* source_lines_numbers;
```